tools/mksurfdata_map/src/mkglacierregionMod.F90

```diff
@@ -104,7 +104,7 @@ subroutine mkglacierregion(ldomain, mapfname, datfname, ndiag, &
 
     max_region = maxval(glacier_region_i)
     call output_diagnostics_index(glacier_region_i, glacier_region_o, tgridmap, &
-         'Glacier Region ID', 0, max_region, ndiag)
+         'Glacier Region ID', 0, max_region, ndiag, mask_src=nint(tgridmap%frac_src), frac_dst=tgridmap%frac_dst)
```

Comment:
Subroutine mkglacierregion returned same answers **with no code changes** when I switched from pointing to
`map_10minx10min_topo_to_0.9x1.25_aave_da_110630.nc`
to
`map_10x10min_nomask_to_0.9x1.25_nomask_aave_da_c191016.nc`

In the above line, however, I had to pass two new arguments required by updated subroutine output_diagnostics_index.

Reply:
Subr. mkglacierregion is unique in that it calls `get_max_indices` for the regridding and returns same answers when I point to the **nomask** as when I point to the **masked** map_ file without requiring code changes.

I introduced the arguments `mask_src` and `frac_dst` to this call to avoid retaining the old `subroutine output_diagnostics_index` that omitted these arguments; however, changing `mask_src=tdomain%mask` and `frac_src=frac_src` messes up the diagnostics, while the tgridmap versions of these work because they equal 1 everywhere.

A more transparent solution would be to set `mask_src=1._r8` and `frac_dst=1._r8`. Let me know if you prefer that or a different option.